nucypher/crypto/powers.py

```diff
@@ -127,42 +126,30 @@ def __init__(self,
         self.__password = password
 
         # Config
-        self.__is_unlocked = False
         self.__blockchain = None
         self.__cache = cache
         self.__activated = False
 
     def __enter__(self):
-        return self.unlock_account()
+        return self.unlock()
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         return self.lock_account()
 
+    def __eq__(self, other):
+        if not isinstance(other, TransactingPower):
+            return False
+        result = bool(self.account == other.account)
```

Comment:
I guess technically we would try to match signers as well (URI?), but Web3Signer being a passthrough would cause issues if we tried that...?

Reply:
Hmm interesting idea.  Ultimately if the same account is controlled, it might be safe enough to assume equality. The only place this equality check is used is in a test in any case so I don't mind removing it either.